frontend/views/containers/user-settings/NotificationSettings.vue

```diff
@@ -1,19 +1,62 @@
 <template lang='pug'>
   .settings-container
     section.card
-      i18n.is-title-2.c-title(tag='h2') Settings
+      i18n.is-title-3.c-title(tag='h2') Browser notifications
+      .c-divider
       .c-subcontent
         .c-text-content
-          i18n.c-smaller-title(tag='h3') Notification Permission
-          i18n.help(tag='p') When you get chat mentions, browser notification will pop up.
+          i18n.c-smaller-title(tag='h3') Allow browser notifications
+          i18n.c-description(tag='p') Get notifications to find out what's going on when you're not on Group Income. You can turn them off anytime.
         label
-          i18n.sr-only Notification Permission
           input.switch(
             type='checkbox'
             name='switch'
-            :checked='granted'
+            :checked='pushNotificationGranted'
             @change='handleNotificationSettings'
           )
+      i18n.is-title-3.c-title(tag='h2') Email notifications
+      .c-divider
+      .c-subcontent
+        .c-text-content
+          i18n.c-smaller-title(tag='h3') Allow email notifications
+          i18n.c-description(tag='p') Know when new proposals are created, their income and get a reminder to vote if they are about to expire.
+        label
+          input.switch(
+            type='checkbox'
+            name='switch'
+            :checked='emailNotificationGranted'
+          )
+      //- i18n.c-name(tag='p') Emails sent
+      //- .c-subcontent
+      //-   .c-text-content
+      //-     i18n.c-smaller-title(tag='h3') Proposals
+      //-     i18n.c-description(tag='p') Know when new proposals are created, their income and get a reminder to vote if they are about to expire.
+      //-   label
+      //-     input.switch(
+      //-       type='checkbox'
+      //-       name='switch'
+      //-       :checked='proposalsGranted'
+      //-     )
+      //- .c-subcontent
+      //-   .c-text-content
+      //-     i18n.c-smaller-title(tag='h3') Payments and contributions
+      //-     i18n.c-description(tag='p') Know when new payments are due or when other members sent contributions your way.
+      //-   label
+      //-     input.switch(
+      //-       type='checkbox'
+      //-       name='switch'
+      //-       :checked='paymentsAndContributionsGranted'
+      //-     )
+      //- .c-subcontent
+      //-   .c-text-content
+      //-     i18n.c-smaller-title(tag='h3') Chat
+      //-     i18n.c-description(tag='p') Get notified when other members sent you messages. You can also change frequency of these emails.
+      //-   label
+      //-     input.switch(
+      //-       type='checkbox'
+      //-       name='switch'
+      //-       :checked='chatGranted'
+      //-     )
```

Comment:
If this block of commented lines are temporarily hiding this feature for a reason, it would be great if the context behind it is stated together here as well. :)